src/Lua/Script.cpp

```diff
@@ -151,8 +151,9 @@ bool Script::get(const std::string& name, bool defaultValue)
     {
         return defaultValue;
     }
-
-    return lua_toboolean(_lua_State, -1);
+    bool ret = lua_toboolean(_lua_State, -1);
```

Comment:
Can we generalize those lines?
Somethings like 

```
#include "LuaBridge/detail/Stack.h"

template <typename T>
T get(lua_State* state) {
    struct StackGuard {
        ~StackGuard() { lua_pop(_state, 1); }

        lua_State* const _state;
    } const guard {state};

    return Stack<T>::get(state);
}

...

return get<bool>(_lua_State);
```
Stack helper from LuaBridge uses luaL_check{number|string} but we already checked type of the stack value and thus we don't need to use lua_to{} API, IMHO